release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,16 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 29 February 2024 - New Feature - Admin - Organization keys added [ID_38944]
+
+The [Admin app](https://admin.dataminer.services) now includes organization keys. A user can create an organization-specific key with certain permissions.
+
+More communication regarding "Organization keys" will follow.
```

Comment:
This is vague and not very helpful, so I think it might be better to just leave this out.

Reply:
Removed this as it's been merged by another PR